wagtail/admin/views/pages/edit.py

```diff
@@ -32,6 +34,81 @@
 )
 
 
+class BaseSidePanel(Component):
+    def __init__(self, page):
+        self.page = page
+
+    class Toggle(Component):
+        def __init__(self, panel):
+            self.panel = panel
+
+        aria_label = gettext_lazy("Toggle")
```

Comment:
As far as I can see this default value isn’t used anywhere, so we should remove it to have one less thing to translate. "Toggle" wouldn’t work as an `aria-label`, it’d need to indicate what is being toggled (which we’re doing well in overrides).